test/test_playbooks/fixtures/domain-3.yml

```diff
@@ -1,242 +1,193 @@
 interactions:
 - request:
-    body: !!python/unicode '{}'
+    body: null
     headers:
-      Accept: ['*/*']
-      Accept-Encoding: ['gzip, deflate']
-      Connection: [keep-alive]
-      Content-Length: ['2']
-      User-Agent: [python-requests/2.20.1]
-      content-type: [application/json]
+      Accept: [application/json;version=2]
     method: GET
-    uri: https://foreman.example.com/katello/api/v2/ping
+    uri: https://192.168.121.223/api/status
```

Comment:
I quite liked the foreman.example.com domain :)

Reply:
blah :)
I'll fix the fixtures at some point, yeah.